packages/dx-react-scheduler/src/plugins/drag-drop-provider.test.tsx

```diff
@@ -288,6 +288,70 @@ describe('DragDropProvider', () => {
       expect(allowResize)
         .toBeCalledWith(deps.template.appointmentTop.data);
     });
+    it('should change Resize render in runtime depending on allowResize prop', () => {
+      const allowResizing = () => true;
+      const deps = {
```

Comment:
Does it necessary for the test?

Reply:
Yes. We need these templates, otherwise they wouldn't exist and Resize component wouldn't be rendered at all